### PR TITLE
feat: add camera functionality

### PR DESCRIPTION
**Pull request by AdityaHarvi**
#### Summary of changes/features added:
- Added camera functionality.
- Added ability to convert bitmaps into strings and back again.

#### Screenshots of UI changes (update with each new commit): 
I'll update the pictures once my phone stops being weird.
image
- This is the screen that opens up when you click on edit-book and click on the image
image
image
image
image

#### Link to referenced github issue: 
https://github.com/CMPUT301F20T05/CriEngine/issues/152

## Pull Request Checklist
- [ ] Unit test written?
- [x] (**For later in project**) Integration tests written?
- [x] Java docs written?

### CODE REVIEW

**Comment by AdityaHarvi** [comment]
I need to get tests written for this still.

**Comment by AdityaHarvi** [comment]
I realized a small issue that could happen. Once the user enters edit mode and make changes, if they pick a new photo and save, then it will revert their text-based changes but keep the new image. o:
ill see if i can fix that

**Review by SamLojpur — APPROVED** [review]
LGTM, I'm going to be taking the bitmap directly though, and I'm basing a branch off of this so don't force push or I'll be really sad

**Comment by ducklin5** [comment]
Lgtm. My only issue is the image gets cropped on small (short) phones. 
image

**Review by ducklin5 — COMMENTED** [review]
Tested and lgtm. 🚀

**Review by ducklin5 — APPROVED** [review]

File: client/app/src/main/java/com/example/criengine/Activities/CameraActivity.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+     *          the main activity.
+     */
+    private AlertDialog confirmDelete() {
+        return new AlertDialog.Builder(this)
+                // set title and message and button behaviors
+                .setTitle("Delete Book")
+                .setMessage("Are you sure you want to delete this image?")
+                .setPositiveButton("DELETE", new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int whichButton) {
+                        // TODO: Image for this book has already been set to null. Push changes to db.
```
Comment (AdityaHarvi) [inline client/app/src/main/java/com/example/criengine/Activities/CameraActivity.java:125]: @SamLojpur This is the final part missing for this PR (database wise). We can wait for you before merging this in

**Comment by AdityaHarvi** [comment]
Important to note that the image will not update immediately when you delete it. I can fix this with passing in another intentExtra though- unless Sam wants to do something with the database to speed it up?

**Review by shadowciaw — APPROVED** [review]

**Review by ducklin5 — APPROVED** [review]

**Pull request merged**